<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1113</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Application demo - FrameECC v0.1.0.0</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="2" column="0">
     <widget class="QGroupBox" name="groupBox">
      <property name="title">
       <string>Console</string>
      </property>
      <layout class="QGridLayout" name="gridLayout_3">
       <item row="1" column="0">
        <widget class="QTextBrowser" name="txtConsole"/>
       </item>
      </layout>
     </widget>
    </item>
    <item row="0" column="0">
     <widget class="QGroupBox" name="groupBox_2">
      <property name="title">
       <string>FPGA actions</string>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout_3">
       <item>
        <widget class="QPushButton" name="btnStart">
         <property name="text">
          <string>START</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="btnStop">
         <property name="text">
          <string>STOP</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label">
         <property name="text">
          <string>Verification status: </string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="imgLed">
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="horizontalSpacer_3">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1113</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu_File">
    <property name="title">
     <string>&amp;File</string>
    </property>
    <addaction name="mnuExit"/>
   </widget>
   <widget class="QMenu" name="menu_About">
    <property name="title">
     <string>&amp;Application</string>
    </property>
   </widget>
   <widget class="QMenu" name="menu_About_2">
    <property name="title">
     <string>&amp;About</string>
    </property>
   </widget>
   <addaction name="menu_File"/>
   <addaction name="menu_About"/>
   <addaction name="menu_About_2"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="mnuExit">
   <property name="text">
    <string>&amp;Exit</string>
   </property>
  </action>
  <action name="mnuReadGroups">
   <property name="text">
    <string>&amp;Read groups</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
